File: c/PROBLEMS/1142.c
#include <stdio.h>

int main(){
    int n, x=1, i;
    scanf("%d", &n);

    for(i=0;i<n;i++){
        printf("%d %d %d PUM\n", x, x+1, x+2);
        x+=4;
    }

}

File: js/PROBLEMS/1142.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

n = parseInt(lines.shift())
x=1
for(i=0;i<n;i++){
    console.log(`${x} ${x+1} ${x+2} PUM`)
    x+=4
}